src/filewatch.erl

```diff
@@ -1,15 +1,49 @@
 -module(filewatch).
--export([start/2, stop/1]).
+-export([start/1,
+         start/2,
+         stop/1]).
+
+-type watch_pair() :: {file:name_all(), term()}.
+-export_type([watch_pair/0]).
+
+-define(DRIVER_NAME, filewatch).
+-define(COOLDOWN_MS,
+        case application:get_env(cooldown_ms) of
```

Comment:
Consider `application:get_env/3` which accepts a default parameter as the third argument.